src/users/cd-create-account.vue

```diff
@@ -258,22 +265,32 @@
       margin-top: 4px;
       font-weight: 300;
     }
-    &__dob-picker {
-      padding-left: 0;
-      max-width: 600px;
+    &__dob-picker-wrapper {
+      max-width: 50%;
```

Comment:
Max-width:50% is terrible on mobile

Reply:
why? as in it stays too big or becomes too small?

Reply:
changed it for 100% on mobile